ddclient.in

```diff
@@ -6428,6 +6438,185 @@ sub nic_hetzner_update {
     }
 }
 
+######################################################################
+## nic_inwx_examples
+######################################################################
+sub nic_inwx_examples {
+    return <<"EoEXAMPLE";
+o 'inwx'
+
+The 'inwx' protocol is an adaption of the 'dyndns2' protocol. This is nescessary
+because INWX uses a custom Update-URL-Format for IPv6-Support:
+
+'https://dyndns.inwx.com/nic/update?myip=<ipaddr>&myipv6=<ip6addr>'
+
+Please note (as of June 2024):
+- INWX does not support updating an IPv6 without including the IPv4 in the
+URL, unless the DynDNS-Account has been turned into IPv6-only manually by their support.
+- Including the IPv6 in the URL without 'myip' or with an empty value for '<ipaddr>'
+will result in the IPv4 (A-Record) beeing reset to '127.0.0.1'.
```

Comment:
These two points seem contradictory.  To me, "does not support" implies an error message, but the second point implies that it silently sets the IPv4 A record to 127.0.0.1.

Reply:
They return NOCHG and set the IPv4 to 127.0.0.1 silently. They do not return an error message.

Reply:
To solve the contradiction: The support of INWX told me, that they can manually turn a dyndns account into an IPv6-only account (No A-Record will be created). I didn't try that though, because it wouldn't solve my problem (I need A + AAAA).